backend/generalhw.pm

```diff
@@ -106,20 +112,35 @@ sub relogin_vnc {
 sub do_start_vm {
     my ($self) = @_;
 
+    $self->truncate_serial_file;
```

Comment:
Don't we call `do_start_vm()` on reboot. And would this mean that each reboot removes everything before?

Reply:
I do not think so. amt, ikvm, ipmi s390x, spvm and svirt backends have it: https://github.com/os-autoinst/os-autoinst/commit/63a2db6ba1617354ba062645a9e15143ed1d2850
@okurz ?

Reply:
sorry, dunno

Reply:
Ok, as this is used in other backends in the same way. Maybe my assumption is just wrong.